packages/react-components/react-breadcrumb-preview/stories/BreadcrumbItem/BreadcrumbItem.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Breadcrumb, BreadcrumbItem, BreadcrumbDivider } from '@fluentui/react-breadcrumb-preview';
+import { bundleIcon, CalendarMonth20Filled, CalendarMonth20Regular } from '@fluentui/react-icons';
+
+const CalendarMonth = bundleIcon(CalendarMonth20Filled, CalendarMonth20Regular);
+
+export const BreadcrumbItemExample = () => {
+  return (
+    <>
+      <Breadcrumb aria-label="breadcrumb-item-icon-example">
+        <BreadcrumbItem icon={<CalendarMonth />}>Item with an icon</BreadcrumbItem>
+        <BreadcrumbDivider />
+        <BreadcrumbItem>Item 1</BreadcrumbItem>
+        <BreadcrumbDivider />
+        <BreadcrumbItem current>Item 2</BreadcrumbItem>
+      </Breadcrumb>
+    </>
+  );
+};
```

Comment:
Based on Ling's comment that non-interactive BreadcrumbItem is only used for showing paths (with a slash divider) then perhaps you delete this story and just have the "BreadcrumbSlashDivider" story to show this usage.

Reply:
I agree. Removed it